requirements/libraries.txt

```diff
@@ -39,12 +39,16 @@ sha3==0.2.1
 unipath>=1.1,<=2.0
 urllib3==1.25.2
 wagtail-autocomplete==0.6
-wagtail-flags==4.2.1
-wagtail-inventory==1.1
-wagtail-sharing==2.2
-wagtail-treemodeladmin==1.2.0
+#wagtail-flags==4.2.1
+#wagtail-inventory==1.1
+#wagtail-sharing==2.2
+#wagtail-treemodeladmin==1.2.0
 
 # These packages are installed from GitHub.
+git+https://github.com/cfpb/wagtail-flags.git@wag210-test
+git+https://github.com/cfpb/wagtail-inventory.git@wag210-test
+git+https://github.com/cfpb/wagtail-sharing.git@wag210-test
+git+https://github.com/cfpb/wagtail-treemodeladmin.git@wag210-test
```

Comment:
As a methodology thing, if we put these in the tox.ini's `future-config` `deps` as well, we could merge this without having any impact on running code.

Reply:
@willbarton That is a really great suggestion I had not thought of, so I have created a `wagtail-future.txt` requirements file to address the future-config wagtail dependencies, then updated `tox.ini` and `requirements/libraries.txt`.

Reply:
That's not quite what I meant. See below.

I'd really rather not expand our requirements files like this.